src/libs/actions/Policy/Policy.ts

```diff
@@ -1355,11 +1356,17 @@ function generateDefaultWorkspaceName(email = ''): string {
     }
     const username = emailParts[0];
     const domain = emailParts[1];
+    const userDetails = PersonalDetailsUtils.getPersonalDetailByEmail(sessionEmail);
+    const displayName = userDetails?.displayName.trim();
```

Comment:
```suggestion
    const displayName = userDetails?.displayName?.trim() ?? '';
```

this should fix the ts check